messages/email_.py

```diff
@@ -18,6 +18,7 @@
 from ._utils import credential_property
 from ._utils import validate_property
 from ._utils import timestamp
+from smtplib import SMTPResponseException
```

Comment:
reorder this to be underneath the "import smtplib" at the top.  Typical import hierarchy from top-to-bottom goes: standard library modules, then third-party modules, then internal package modules.